AsyncDisplayKit/ASTextNode.mm

```diff
@@ -379,7 +381,22 @@ - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
 {
   // Offset the text origin by any shadow padding
   UIEdgeInsets shadowPadding = [self shadowPadding];
-  CGPoint textOrigin = CGPointMake(self.bounds.origin.x - shadowPadding.left, self.bounds.origin.y - shadowPadding.top);
+  CGPoint textOrigin = CGPointMake(self.bounds.origin.x - shadowPadding.left, 0.0f);
+  
+  switch (_verticalTextAlignment)
+  {
+    case ASTextNodeVerticalTextAlignmentTop:
+      textOrigin.y = self.bounds.origin.y - shadowPadding.top;
+      break;
+    case ASTextNodeVerticalTextAlignmentCenter:
+      textOrigin.y += (self.bounds.size.height - [self _renderer].size.height) / 2.0f;
+      break;
+    case ASTextNodeVerticalTextAlignmentBottom:
+      textOrigin.y = self.bounds.size.height - shadowPadding.top - shadowPadding.bottom - [self _renderer].size.height;
```

Comment:
Missing a break statement on this one!  Technically doesn't cause a bug, but needs to be added.